src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/Encoder.cs

```diff
@@ -16,6 +16,21 @@ public sealed class Encoder
         public static readonly Encoder LuminanceTable = new Encoder(new Guid(unchecked((int)0xedb33bce), unchecked((short)0x0266), unchecked((short)0x4a77), new byte[] { 0xb9, 0x04, 0x27, 0x21, 0x60, 0x99, 0xe7, 0x17 }));
         public static readonly Encoder ChrominanceTable = new Encoder(new Guid(unchecked((int)0xf2e455dc), unchecked((short)0x09b3), unchecked((short)0x4316), new byte[] { 0x82, 0x60, 0x67, 0x6a, 0xda, 0x32, 0x48, 0x1c }));
         public static readonly Encoder SaveFlag = new Encoder(new Guid(unchecked((int)0x292266fc), unchecked((short)0xac40), unchecked((short)0x47bf), new byte[] { 0x8c, 0xfc, 0xa8, 0x5b, 0x89, 0xa6, 0x55, 0xde }));
+        
+        /// <summary>
+        /// An <see cref="Encoder" /> object that is initialized with the globally unique identifier for the color space category.
+        /// </summary>
+        public static readonly Encoder ColorSpace = new Encoder(new Guid(unchecked((int)0xae7a62a0), unchecked((short)0xee2c), unchecked((short)0x49d8), new byte[] { 0x9d, 0x07, 0x1b, 0xa8, 0xa9, 0x27, 0x59, 0x6e }));
+        
+        /// <summary>
+        /// An <see cref="Encoder" /> object that is initialized with the globally unique identifier for the image items category.
+        /// </summary>
+        public static readonly Encoder ImageItems = new Encoder(new Guid(unchecked((int)0x63875e13), unchecked((short)0x1f1d), unchecked((short)0x45ab), new byte[] { 0x91, 0x95, 0xa2, 0x9b, 0x60, 0x66, 0xa6, 0x50 }));
+        
+        /// <summary>
+        /// An <see cref="Encoder" /> object that is initialized with the globally unique identifier for the save as CMYK category.
```

Comment:
I think we try to use the full name even if it is not needed, but not sure. @carlossanlop ?

Reply:
Do you mean using `<see cref="Encoder" />`? No, it's not needed in triple slash comments. If the API was not recognized within this context, VS would give you an error. But in this case, we recognize the `Encoder` type.
When the project gets built, the triple slash comments get converted to the xml files, which will automatically convert types to their full name.

Reply:
Using the `T:` prefix results in build errors: `error CA1200: Avoid using cref tags with a prefix`